src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,79 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task with a deadline.
+ * A Deadline task has a description and a specific date and time
+ * by which it needs to be completed.
+ */
+public class Deadline extends Task{
```

Comment:
Take note of the missing single space between `Task` and `{`.

Similarly for a few other classes as well.